### PR TITLE
Clarify the query cache default behavior.

### PR DESCRIPTION
**Pull request by Vinai**
## Purpose of this pull request

This PR clarifies the default behavior for query caching.

## Affected DevDocs pages

* https://devdocs.magento.com/guides/v2.3/graphql/develop/create-graphqls-file.html

## Links to Magento source code

https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/GraphQlCache/Model/CacheableQuery.php#L71-L76

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/develop/create-graphqls-file.md [hunk 1]
```diff
@@ -127,8 +127,14 @@ Define cachable queries in the following manner:
...
-Queries that explicitly cannot be cached are designated with the `@cache(cacheable: false)` directive.
+A query without a `cacheIdentity` will not be cached.
+
+To disable caching for queries declared in another module with a `cacheIdentity` class, the `@cache(cacheable: false)` directive can be used.
+This `cacheable` argument is intended to disable caching for queries that are defined in another module.
 
 `@cache(cacheable: false)`
 
+Specifying `@cache(cacheable: false)` or `@cache(cacheable: true)` on a query without a `cacheIdentity` class has no effect: the query will not be cached.
+If a query should **no**t be cached, no `@cache` directive at all should be specified. Specifying `@cache(cacheable: false)`  is superfluous when no `cacheIdentity` is present.
```
Comment (keharper) [inline guides/v2.3/graphql/develop/create-graphqls-file.md]:
```suggestion
If a query should **not** be cached, do not specify the `@cache` directive. Specifying `@cache(cacheable: false)`  is superfluous when no `cacheIdentity` is present.
```

**Comment by Vinai** [comment]
Thanks!

**Comment by keharper** [comment]
Thanks @Vinai I see that you asked the question on the Community Engineering #graphql Slack channel, found and posted the results, and then opened this PR. I appreciate your devotion.

**Pull request merged**